src/LIBSVM.jl

```diff
@@ -298,7 +297,7 @@ function EpsilonSVR(
         ,gamma
         ,epsilon
         ,cost
-        ,cache_size
+        ,cachesize
```

Comment:
Generally, we try to stick with variable names that coincide with the source package documentation. I guess you checked this?